android/java/res/drawable/rewards_tipping_radio_button_selector.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:state_checked="false" android:drawable="@drawable/rewards_tipping_radio_button_normal" />
+    <item android:state_checked="true" android:drawable="@drawable/rewards_tipping_radio_button_selected" />
+</selector>
```

Comment:
Add new line at the end as per chromium guidelines

Reply:
done